airflow/configuration.py

```diff
@@ -92,7 +92,7 @@ def _default_config_file_path(file_name: str):
     return os.path.join(templates_dir, file_name)
 
 
-def default_config_yaml() -> dict:
+def default_config_yaml() -> List[dict]:
```

Comment:
Is this right? PyYAML documentation says `safe_load` only loads the first document, which would be `dict`.

Reply:
https://pyyaml.org/wiki/PyYAMLDocumentation

> `safe_load(stream)` parses the given stream and returns a Python object constructed from for the first document in the stream. If there are no documents in the stream, it returns `None`. `safe_load` recognizes only standard YAML tags and cannot construct an arbitrary Python object.

Reply:
Yeah, it is a list:
https://github.com/apache/airflow/blob/main/airflow/config_templates/config.yml

PyYAML is referring to YAML documents: https://pyyaml.org/wiki/PyYAMLDocumentation#documents

We only have one, but you could have more than one in the steam, e.g:
```
---
- a:b
---
- c:d
```

Interestingly, for me `safe_load` actually raises when it sees more than 1 document 🤷‍♂️